app/src/main/java/com/example/happymeals/Recipe.java

```diff
@@ -0,0 +1,105 @@
+package com.example.happymeals;
+
+import com.google.firebase.firestore.DocumentReference;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+/**
+ * @author jeastgaa
+ * @version 1.00.01
+ * @see DatabaseObject The parent object for this class which allows it to be stored inside the
+ * firebase databse using the {@link FireStoreManager}
+ * This class represents recipes that can be loaded inside the application and shown to the
+ * end user.
+ */
+public class Recipe extends DatabaseObject {
+
+    private double cookTime;
```

Comment:
Need comments attribute see user story [US 02.01.01 Add ability to add recipes#9](https://github.com/CMPUT301F22T32/Happy-Meals/issues/9)